Week3-0422/MatchCardsGame/src/Pages/layout/Header.jsx

```diff
@@ -0,0 +1,97 @@
+import styled from "styled-components";
+import { BigButton } from "../../components/Buttons";
+import { useEffect } from "react";
+import { ProgressBar } from "react-progressbar-fancy";
+
+const Wrapper = styled.header`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  height: 25rem;
+  background-color: ${(props) => props.theme.headerColor};
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.2);
+`;
+
+const MainCenterBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  width: 60%;
+
+  h1 {
+    font-size: 4.5rem;
+    color: ${(props) => props.theme.textColor};
```

Comment:
theme을 쓰고 싶었던 거였다면! 보통은 이렇게 작성한답니당
```suggestion
    color: ${({theme}) =>theme.textColor};
```